src/busio.py

```diff
@@ -16,10 +16,8 @@ def __init__(self, scl, sda, frequency=400000):
 
     def init(self, scl, sda, frequency):
         self.deinit()
-        if board_id == "raspi_3" or board_id == "raspi_2":
-            from adafruit_blinka.microcontroller.raspi_23.i2c import I2C as _I2C
-        if board_id == "beaglebone_black":
-            from adafruit_blinka.microcontroller.raspi_23.i2c import I2C as _I2C
+        if board_id == "raspi_3" or board_id == "raspi_2" or board_id == "beaglebone_black":
```

Comment:
How about stashing all these in a tuple and then doing something like:
```python
if board_id in LINUX_BOARDS:
```

Reply:
ill do that long term but i need to redo the platform detection code furst

Reply:
coolio.